### DISCUSSION TITLE
Démarrage domotique chauffage

### DISCUSSION THREAD

**Discussion by Adylan60** [haade-administrator/haade.fr#59] [Q&A]
Bonjour, 

Je démarre la domotique avec les chauffage, je suis avec une chaudière à gaz du coup j'ai commander un contacteur sec Nodon.
J'ai aussi pris des tête thermostatique Sonoff et des capteur température Sonoff.

Mais là arrive mon problème, je ne sais pas si je prend la home assistance green avec la clé sky connect  ou une box ihost Sonoff ou autre.

Pouvez vous m'aider a faire mon choix et comprendre la différence.

**Comment by haade-administrator** [comment]
la box sonoff ihost est relativement ouverte aux marques, mais très limité par rapport à home assistant, automatisation et blueprint
Si tu veux faire le max d'économies je te conseil vivement la home assistant green et skyconnect ainsi tu pourras faire comme august des économies. Un article complet sur l'installation: https://haade.fr/fr/blog/economise-du-chauffage-grace-a-la-domotique-ca-marche
